scripts/common/get_storage_plugin.sh

```diff
@@ -23,7 +23,9 @@ __version__="1.0"
 
 # Get the storage database plugin from the Storage microservice cache file
 get_storage_plugin() {
-
+if [ "${FOGLAMP_ROOT}" ]; then
```

Comment:
In a push to make this more bullet proof I would suggest we do the following

if [ "${FOGLAMP_ROOT}" ]; then
    $FOGLAMP_ROOT/scripts/services/storage --plugin | cut -d' ' -f1
elif [ -x scripts/services/storage ]; then
    scripts/services/storage --plugin | cut -d' ' -f1
elif [ -x /usr/local/foglamp/scripts/services/storage ]; then
    /usr/local/foglamp/scripts/services/storage --plugin | cut -d' ' -f1
else
        .... LOG ERROR and force exit ....
fi

Reply:
Added log err.
/usr/local/foglamp/ case not added.
The script is called via FOGLAMP_ROOT by foglamp scripts and without it when called by make install